vm-image-service/README.md

```diff
@@ -49,14 +27,37 @@ The UI allows users to execute all necessary workflows:
         - Show details for the Content Library
     - Grid action menu
         - View the YAML for a Content Library
+- Publish VM to a Content Library
 
 ## List all Virtual Machine Images
+![VM Image Card](source/images/vm-image-card.png "VM Image Card")
+VM Image service can be accessed via a card on the namespace summary view. If VM Image service is available for the current namespace then the above shown card will appear in the namespace summary view. Check out the support section in this document.
+
+![VM Image List View](source/images/vm-image-image-list.png "VM Image List View")
+In the above picture, we are looking at the list of VirtualMachine Images which are available for the current namespace. Namespaces are not directly associated with VM Images rather transitively associated through Content Libraries. The above list view is an aggregation of all the VM Images across all Content Libraries that are associated with the current namespace.
 
 ### Actions
+![VM Image Delete](source/images/vm-image-delete.png "VM Image Delete")
+The only user action available for consumption users to perform on a VM Image is "Delete" a VM Image from a namespace-scoped Content Library that has been marked as writable. Delete action will be left unavailable for VM Images which belong to cluster-scoped and read-only libraries.
+
 ### Details
+![VM Image Summary](source/images/vm-image-summary.png "VM Image Summary")
+Clicking an image name in the VM Image list view will navigate to a detailed summary view for the selected VM Image. On this view, you will be able to view all the details of VM Image, check health status, and view the list of latest Kubernetes events for the given image.
+
 
-## List all Content Library contents
+## List all Content Library resources
+![Content Library List](source/images/vm-image-library-list.png "Content Library List")
+Content Library list view displays the list of content library resources associated with the current namespace. There are two kinds of library resources, with regards to the scope, *global scoped* and *namespace scoped*. 
```

Comment:
Maybe

There are two kinds of library resources with regards to the scope: *global scoped* and *namespace scoped*.

Reply:
See note above: we should explain that earlier.

Reply:
Thank you.
I have added a new section explaining different kind of scopes towards the beginning of the document.